test-packages/ts-template-imports-app/app/components/Greeting.gts

```diff
@@ -13,10 +14,12 @@ export default class Greeting extends Component<GreetingSignature> {
   </template>
 }
 
-function repeat(value: string, times: number): string {
-  return Array(times).fill(value).join('');
-}
-
 const Bang: TOC<{ Args: { times: number } }> = <template>
   {{repeat '!' @times}}
 </template>
+
+declare module '@glint/environment-ember-loose/registry' {
```

Comment:
why was this added? it's only needed for loose mode

Reply:
https://github.com/typed-ember/glint/pull/620/files#r1331742697

Reply:
I thought it would be a good idea to incorporate loose mode to make sure we have decent coverage for this scenario since in today's apps using .gts files, they will still have route templates.